renuo-bin-check.gemspec

```diff
@@ -0,0 +1,32 @@
+# coding: utf-8
+lib = File.expand_path('../lib', __FILE__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require 'renuo/bin-check/version'
+
+Gem::Specification.new do |spec|
+  spec.name = 'renuo-bin-check'
+  spec.version = Renuo::BinCheck::VERSION
+  spec.authors = ['Zora Fuchs']
+  spec.email = ['zora.fuchs@renuo.ch']
+
+  spec.summary = 'The Renuo bin-check automates running programms to check code quality of a ruby application. For faster runtime it makes use of cashing and parallel execution.'
+  spec.description = 'The Renuo bin-check automates running programms to check code quality of a ruby application. For faster runtime it makes use of cashing and parallel execution.'
+  spec.homepage = 'https://github.com/renuo/renuo-bin-check'
+  spec.license = 'MIT'
+
+  spec.files = `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) }
+  spec.bindir = 'exe'
+  spec.executables = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
+  spec.require_paths = ['lib']
+  spec.test_files    = `git ls-files -- {test,spec,features}/*`.split("\n")
+
+  spec.add_development_dependency 'rubocop',          '>= 0.39.0'
```

Comment:
Nice! :) you may also be a bit more specific here, e.g. like https://github.com/renuo/renuo-cms-rails/blob/master/renuo_cms_rails.gemspec#L28